### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by sithlord48**
Change to english src strings.

### CODE REVIEW

**Comment by sithlord48** [comment]
the japanese ts file needs to be updated

File: Window.cpp [hunk 1]
```diff
@@ -1017,12 +1017,12 @@ void Window::exporter()
 		index = path.lastIndexOf('/');
 		Config::setValue("exportPath", index == -1 ? path : path.left(index));
 		break;
-	case 1:	out = tr("L'archive Lgp est inaccessible");break;
-	case 2:	out = tr("Erreur lors de l'ouverture du fichier");break;
-	case 3:	out = tr("Impossible de créer le nouveau fichier");break;
-	case 4:	out = tr("Pas encore implémenté !");break;
+	case 1:	out = tr("lgp archive is inaccessible");break;
```
Comment (myst6re) [inline Window.cpp]: Lgp

File: Window.cpp [hunk 2]
```diff
@@ -1437,14 +1436,14 @@ void Window::about()
 
 	font.setPointSize(8);
 
-	QLabel desc2(tr("Par myst6re<br/><a href=\"https://github.com/myst6re/makoureactor/\">github.com/myst6re/makoureactor</a><br/><br/>Merci à :<ul style=\"margin:0\"><li>Squall78</li><li>Synergy Blades</li><li>Akari</li><li>Asa</li><li>Aali</li></ul>"), &about);
+	QLabel desc2(tr("By myst6re<br/><a href=\"https://github.com/myst6re/makoureactor/\">github.com/myst6re/makoureactor</a><br/><br/>Thanks to :<ul style=\"margin:0\"><li>Squall78</li><li>Synergy Blades</li><li>Akari</li><li>Asa</li><li>Aali</li></ul>"), &about);
```
Comment (myst6re) [inline Window.cpp]: You can remove the space bewteen Thanks to and :

File: widgets/TextManager.cpp [hunk 3]
```diff
@@ -429,7 +429,7 @@ void TextManager::delText()
 	if(!item) return;
 	int row=item->data(Qt::UserRole).toInt();
 	if(usedTexts.contains(row)) {
-		QMessageBox::StandardButton rep = QMessageBox::warning(this, tr("Texte utilisé dans les script"), tr("Ce texte est utilisé par un ou plusieurs scripts de cet écran.\nLe supprimer remplacera les appels à ce texte par des appels au texte qui suit.\nÊtes-vous sûr de vouloir continuer ?"), QMessageBox::Yes | QMessageBox::Cancel, QMessageBox::Cancel);
+		QMessageBox::StandardButton rep = QMessageBox::warning(this, tr("Text used in scripts"), tr("This text is used by one or more scripts on this field..\nRemove will replace calls to this text with calls to the text that follows.\nAre you sure you want to continue?"), QMessageBox::Yes | QMessageBox::Cancel, QMessageBox::Cancel);
```
Comment (myst6re) [inline widgets/TextManager.cpp]: Two dots here

**Comment by myst6re** [comment]
Great job!

**Pull request merged**